.buildkite/pipeline.yml

```diff
@@ -2,7 +2,7 @@
 common_params:
   # Common plugin settings to use with the `plugins` key.
   - &common_plugins
-    - automattic/bash-cache#2.8.0
+    - automattic/bash-cache#mokagio/genstring-in-ci
```

Comment:
Will need to push a new commit or rewrite this one to point to the next `bash-cache` version once https://github.com/Automattic/bash-cache-buildkite-plugin/pull/36 lands.